README.md

```diff
@@ -1 +1,13 @@
-# vagrant-is
\ No newline at end of file
+# WSO2 Identity Server Vagrant Resources
+
+This repository contains the following resources:
+
+- Automated resources for Vagrant box creation
+
+Resources which automate the Vagrant box creation process are contained here. Users can define
+box specifications based on which the Vagrant boxes are generated.
+
+- Vagrantfile for WSO2 Identity Server single node deployment with Analytics support
```

Comment:
This can be simply 
"Vagrantfile for WSO2 Identity Server deployment with Analytics"

Reply:
+1